### PR TITLE
Make products client side

### PR DESCRIPTION
**Pull request by Bilalm354**
todo: get size and colour working and check that they pass through to the order

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_ENV="development"
+REPLICATE_API_KEY="r8_SkzHkBuWIcaTmJg8aSkUbDjFP1LnTrI2B4upN"
```
Comment (haffi96) [inline .env.development:2]: remove api key?

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# disabled because doesn't run on ci yet and don't want to have a failing check
+name: Playwright Tests
+on:
+    push:
+        branches: [main, master]
+    pull_request:
+        branches: [main, master]
```
Comment (haffi96) [inline .github/workflows/playwright.yml:7]:
```suggestion
        branches: [main]
    pull_request:
        branches: [main]
```

File: .github/workflows/playwright.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+              with:
+                  node-version: lts/*
+        # - name: Install dependencies
+        #   run: npm ci
+        # - name: Install Playwright Browsers
+        #   run: npx playwright install --with-deps
+        # - name: Run Playwright tests
+        #   run: npx playwright test
+        # - uses: actions/upload-artifact@v4
+        #   if: always()
```
Comment (haffi96) [inline .github/workflows/playwright.yml:24]: always just returns true? So can remove it?

Comment (Bilalm354) [reply .github/workflows/playwright.yml:24]: was autogenerated

Comment (Bilalm354) [reply .github/workflows/playwright.yml:24]: but yeah i guess

File: .env.development (hunk 1 above)
Comment (Bilalm354) [reply .env.development:2]: done

**Review by haffi96 — APPROVED** [review]

**Pull request merged**